dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/relationship/RelationshipsExportController.java

```diff
@@ -70,6 +73,7 @@
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.google.common.collect.ImmutableMap;
 
+@OpenApi.EntityType( Relationship.class )
 @OpenApi.Tags( "tracker" )
 @RestController
 @RequestMapping( produces = APPLICATION_JSON_VALUE, value = RESOURCE_PATH + "/"
```

Comment:
I was wondering if the `value` here should end with a `/` since the usual approach is that the method level path part starts with a `/`.

Reply:
We discussed this and this seems to be correct 😄 The top level mapping should not end in `/` as it allows requests like `/enrollments` and `/enrollments/` to be processed by `getEnrollments`. The endpoints returning single entities have the `/` prefix in their mapping `/{uid}`